Sources/CollectionViewManager.swift

```diff
@@ -10,6 +10,7 @@ open class CollectionViewManager: NSObject {
     
     public typealias SectionItem = CollectionViewSectionItem
     public typealias CellItem = CollectionViewCellItem
+    public typealias ViewItem = CollectionViewReusableViewItem
```

Comment:
Based on above examples it should be called
```suggestion
    public typealias ReusableViewItem = CollectionViewReusableViewItem
```